optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/statistic/ProblemStatisticType.java

```diff
@@ -62,4 +64,8 @@ public boolean hasScoreLevels() {
                 || this == STEP_SCORE;
     }
 
+    public static List<ProblemStatisticType> defaultMetricList() {
```

Comment:
Considering that this returns `ProblemStatisticType`, it should probably be called `defaultProblemStatisticTypeList()`.
Also, considering that this is an enum, we can make use of `EnumSet`, therefore making this `defaultProblemStatisticTypeSet()`.
But seeing as this is already on `ProblemStatisticType`, I think that just calling it `defaultSet()` is acceptable.

Reply:
Not sure about turning it into a set; are usages of this method is used for lists, which will make them more complex for little/no gain. I don't think `ProblemStatisticType` should be in the name since it a static method on `ProblemStatisticType`, which duplicates the information (unless it is statically imported). `defaultList` could work.

Reply:
Good point on the `Set`. Change the name, please, and we should be done.

Reply:
One thought just crossed my mind: `List.of(...)` returns an *immutable* list. Is that what we expect in the API? If the users read the list from the config, and attempt to insert, they'll get an exception.

Reply:
On the other hand, this list here pretty much has to be immutable, because we do not want people to change the defaults. So this mutability issue must be resolved at the call site in the config.

Reply:
Agree, mutable defaults are a horrible idea.